README.md

```diff
@@ -39,6 +39,34 @@ vsql or admintools, etc. and uses the lastest VerticaPy version.
    ```
    make uninstall
    ```
+## VerticaPy Development
+
+In order to use VerticaPyLab for development of VerticaPy, the following changes can be made to create a VerticaPyLab image that does not have VerticaPy installed:
+
+1. ``cd`` into VerticaPyLab/docker-verticapy. Edit the ``requirements.txt`` file and remove "verticapy" from the list.
+2. If you also want the installation to be light and remove unnecessary extensions, open the Dockerfile in the same directory and comment out from line 53 to 63 [This is optional]
```

Comment:
Would it be better to show those lines? Because if the Dockerfile is updated the line numbers might be different.